apps/web/tests/setup/example.ts

```diff
@@ -25,13 +25,15 @@ class Example {
   readonly firstName: Field
   readonly email: Field
   readonly password: Field
+  readonly favColor: Field
   readonly button: Locator
 
   constructor(page: Page) {
     this.page = page
     this.firstName = this.field('firstName')
     this.email = this.field('email')
     this.password = this.field('password')
+    this.favColor = this.field('favColor')
```

Comment:
I had missed this one: please do not add this field to the default test example and add it only to your test file.

Reply:
I'm sorry, yes i should've not done that.
It's fixed.